### ISSUE TITLE
COMException (0x800706BA): The RPC server is unavailable. (Exception from HRESULT: 0x800706BA)

### ISSUE THREAD

**Issue by vegardlarsen** [digitalcreations/MaxTo#755]
Originally reported by @SalmanRavoof in #753:

I get a similar error, though it says something else. 

```
System.Runtime.InteropServices.COMException (0x800706BA): The RPC server is unavailable. (Exception from HRESULT: 0x800706BA)
   at MaxTo.Core.Native.VirtualDesktops.Interop.IVirtualDesktopNotificationService.Unregister(UInt32 cookie)
   at MaxTo.Core.Native.VirtualDesktops.VirtualDesktop.set_EnableRaisingEvents(Boolean value)
   at MaxTo.Core.Recipes.Triggers.VirtualDesktopTriggerController.Dispose()
   at MaxTo.Core.Events.EventDriver.Dispose()
   at MaxTo.Core.MainForm.Dispose(Boolean disposing)
   at System.Windows.Forms.Form.WmClose(Message& m)
   at MaxTo.Core.Native.HiddenWindowMessageSource.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

**Comment by vegardlarsen** [comment]
@SalmanRavoof Which version of Windows and MaxTo are you running?

**Comment by SalmanRavoof** [comment]
@vegardlarsen 
Windows 21H1 (OS Build 19043.1110)
MaxTo 2.2.0

**Comment by vegardlarsen** [comment]
This issue has been fixed for version 2.2.1.
